lib/google_search_data_viewer_worker/keywords/helpers/job_creation.ex

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleSearchDataViewerWorker.Keywords.JobCreationHelper do
```

Comment:
IMO this content of this module can be put in `GoogleSearchDataViewerWorker.Keywords` because the function belongs to the _Keywords_ context

Reply:
I initially thought the same, but as IMO it was unrelated to the business / domain logic but with the creation of asynchronous `Oban` jobs I thought it more accurately belonged to the `worker` context

Reply:
Yes, what I meant is to have a `Keywords` context under the `GoogleSearchDataViewerWorker` scope. Because the function is to create keyword jobs. 

What do you think?

Reply:
Thanks Micky for clarificaiton.

Unless I am missing something I think it is already listed under GoogleSearchDataViewerWorker/Keywords. It might look a bit confusing because that's the only location I have files, ie none are listed outside of  GoogleSearchDataViewerWorker/Keywords

Reply:
I think this module can be move from `GoogleSearchDataViewerWorker.Keywords.JobCreationHelper` to `GoogleSearchDataViewerWorker.Keywords` and for the file it can locate in `lib/google_search_data_viewer_worker/keywords.ex`

Reply:
Would it be OK if I did this change in the following Chore PR where I also address Thanh's comments about context name changes etc? Or do you suggest I change these in this PR